File: go/wam/6110.go
const WamProtobufValidationError = 6110 // channel: regular

type ProtobufValidationErrorEvent struct {
	ProtobufValidationDropped bool `wam:"protobuf_validation_dropped"` // field 1
	ProtobufValidationErrorMessage string `wam:"protobuf_validation_error_message"` // field 2
	ProtobufValidationExpression string `wam:"protobuf_validation_expression"` // field 3
	ProtobufValidationFlow PROTOBUFVALIDATIONFLOW `wam:"protobuf_validation_flow"` // field 4
	ProtobufValidationPath string `wam:"protobuf_validation_path"` // field 5
	ProtobufValidationRuleId string `wam:"protobuf_validation_rule_id"` // field 6
	ProtobufLegacyValidationDropped bool `wam:"protobuf_legacy_validation_dropped"` // field 7
	ProtobufCorrelationOutcome PROTOBUFCORRELATIONOUTCOME `wam:"protobuf_correlation_outcome"` // field 8
	ProtobufValidationContext int64 `wam:"protobuf_validation_context"` // field 9
	ProtobufValidationThumbnailFormat string `wam:"protobuf_validation_thumbnail_format"` // field 10
}

File: rust/wam/6110.rs
pub const WAM_PROTOBUF_VALIDATION_ERROR: u32 = 6110;

#[derive(Debug, Default)]
pub struct ProtobufValidationErrorEvent {
    /// field 1, wire `protobuf_validation_dropped`
    pub protobuf_validation_dropped: Option<bool>,
    /// field 2, wire `protobuf_validation_error_message`
    pub protobuf_validation_error_message: Option<String>,
    /// field 3, wire `protobuf_validation_expression`
    pub protobuf_validation_expression: Option<String>,
    /// field 4, wire `protobuf_validation_flow`
    pub protobuf_validation_flow: Option<PROTOBUFVALIDATIONFLOW>,
    /// field 5, wire `protobuf_validation_path`
    pub protobuf_validation_path: Option<String>,
    /// field 6, wire `protobuf_validation_rule_id`
    pub protobuf_validation_rule_id: Option<String>,
    /// field 7, wire `protobuf_legacy_validation_dropped`
    pub protobuf_legacy_validation_dropped: Option<bool>,
    /// field 8, wire `protobuf_correlation_outcome`
    pub protobuf_correlation_outcome: Option<PROTOBUFCORRELATIONOUTCOME>,
    /// field 9, wire `protobuf_validation_context`
    pub protobuf_validation_context: Option<i64>,
    /// field 10, wire `protobuf_validation_thumbnail_format`
    pub protobuf_validation_thumbnail_format: Option<String>,
}